### PR TITLE
ES6 Migration: server/adapters/SchedulingDefault.js

### PR DESCRIPTION
**Pull request by billfienberg**
ES6 Migration: server/adapters/SchedulingDefault.js

issue #9589

- Replaces `var`s with `let`/`const`s
- Use object shorthand (Replace `{err: err}` with `{err}`)
- Replace concatenated strings with template literals

### CODE REVIEW

File: core/server/adapters/scheduling/SchedulingDefault.js [hunk 1]
```diff
@@ -53,19 +53,20 @@ SchedulingDefault.prototype.unschedule = function (object) {
  * because allJobs is a sorted list, we don't have to iterate over all jobs, just until the offset is too big
  */
 SchedulingDefault.prototype.run = function () {
-    var self = this,
-        timeout = null,
+    const self = this;
+    let {isRunning} = this;
```
Comment (kirrg001) [inline core/server/adapters/scheduling/SchedulingDefault.js]: 🤔 What is the reason you are not accessing `this.isRunning`?

File: core/server/adapters/scheduling/SchedulingDefault.js [hunk 2]
```diff
@@ -147,30 +150,31 @@ SchedulingDefault.prototype._deleteJob = function (object) {
  * we don't want to use process.nextTick, this would block any I/O operation
  */
 SchedulingDefault.prototype._execute = function (jobs) {
-    var keys = Object.keys(jobs),
+    const keys = Object.keys(jobs),
         self = this;
```
Comment (kirrg001) [inline core/server/adapters/scheduling/SchedulingDefault.js]: There are a couple of `self` usages in this file. Can they be removed when using arrow functions?

**Comment by kirrg001** [comment]
@billfienberg Thanks for your time. I left two smaller comments.

File: core/server/adapters/scheduling/SchedulingDefault.js (hunk 1 above)
Comment (billfienberg) [reply core/server/adapters/scheduling/SchedulingDefault.js]: I tried to completely refactor out `self`, but some integration tests started failing. Therefore, I left `self` in for now. I'd like to revisit it in the future, but I didn't want that to stop from me submitting what I had done so far.

File: core/server/adapters/scheduling/SchedulingDefault.js (hunk 2 above)
Comment (billfienberg) [reply core/server/adapters/scheduling/SchedulingDefault.js]: I tried to completely refactor out self, but some integration tests started failing. Therefore, I left self in for now. I'd like to revisit it in the future, but I didn't want that to stop from me submitting what I had done so far.

Comment (kirrg001) [reply core/server/adapters/scheduling/SchedulingDefault.js]: Ok 👍

File: core/server/adapters/scheduling/SchedulingDefault.js (hunk 1 above)
Comment (kirrg001) [reply core/server/adapters/scheduling/SchedulingDefault.js]: I don't see a reason to change `this.running`. This is IMO independent from the ES6 task/issue. It just changes the coding style. Could you please revert? Thanks!

**Review by kirrg001 — CHANGES_REQUESTED** [review]
See comment :)

File: core/server/adapters/scheduling/SchedulingDefault.js (hunk 1 above)
Comment (billfienberg) [reply core/server/adapters/scheduling/SchedulingDefault.js]: I will revert it.

Comment (billfienberg) [reply core/server/adapters/scheduling/SchedulingDefault.js]: It's been reverted.
https://github.com/TryGhost/Ghost/pull/9696/commits/91239fd731e9d0278c08e9515f5112b774f2dd5a

**Review by kirrg001 — APPROVED** [review]

**Pull request merged**